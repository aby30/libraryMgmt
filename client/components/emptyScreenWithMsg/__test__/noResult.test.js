import React from 'react';
import { render, cleanup } from '@testing-library/react';
import NoResult from '../index';

afterEach(cleanup);

describe('No Result screen', () => {

    it('renders', () => {
        const { asFragment } = render(<NoResult textToShow='No results text here' />);
        expect(asFragment()).toMatchSnapshot();
    })

})