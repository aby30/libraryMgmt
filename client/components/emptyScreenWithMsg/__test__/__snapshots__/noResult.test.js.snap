// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Result screen renders 1`] = `
<DocumentFragment>
  <div
    class="noResult"
  >
    <div
      class="noResult__inner"
    >
      No results text here
    </div>
  </div>
</DocumentFragment>
`;
